.github/workflows/ci.yml

```diff
@@ -199,8 +252,44 @@ jobs:
         run: echo "TRANSFORMERS_TAG=$(mvn help:evaluate -Dexpression=dependency.alfresco-transform-core.version -q -DforceStdout)" >> $GITHUB_ENV
       - name: "Set up the environment"
         run: docker-compose -f ./scripts/ci/docker-compose/docker-compose.yaml --profile ${{ matrix.compose-profile }} up -d
+      - name: "Prepare Report Portal"
+        uses: Alfresco/alfresco-build-tools/.github/actions/reportportal-prepare@v5.1.0
+        id: rp-prepare
+        with:
+          rp-launch-prefix: ${{ env.RP_LAUNCH_PREFIX }} - ${{ matrix.testSuite }}
+          rp-token: ${{ secrets.REPORT_PORTAL_TOKEN }}
```

Comment:
Please double-check that if this secret is not available as it happens for DependaBot PRs then the build will still succeed and run all necessary tests. you can easily test this by temporarily changing this line to `rp-token: "{{ secrets.SECRET_THAT_DOES_NOT_EXIST }}`

Reply:
Checked, all jobs still works correctly: https://github.com/Alfresco/alfresco-community-repo/actions/runs/8457827488